app/assets/stylesheets/components/_login-banner.scss

```diff
@@ -0,0 +1,36 @@
+.login-banner {
+  background-color: $color-gray-cool-light;
+  color: $color-primary;
+  padding: 1rem 0;
+}
+
+.homepage-login-banner .login-banner {
```

Comment:
This may be a case where you could nest this selector and its contents beneath the `.login-banner` styles above.

```scss
.login-banner {
  background-color: …
  …

  .homepage-login-banner & {
    …
  }
}
```

Remember last week our discussion about using the parent select (`&`). You could (if you want) refactor this using that selector. It's totally up to you, though.